modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -4,66 +4,167 @@
+
+server.discovery.advertised_address=server01.example.com:5000
+dbms.cluster.discovery.endpoints=cluster01.example.com:5000
+
+dbms.cluster.discovery.version=V1_ONLY
+----
+======
+[role=include-with-discovery-service-v2 label--new-5.23]
+======
+[source, parameters]
+----
+dbms.cluster.discovery.resolver_type=DNS
+
+server.cluster.advertised_address=server01.example.com:6000
+dbms.cluster.discovery.v2.endpoints=cluster01.example.com:6000
+dbms.cluster.discovery.version=V2_ONLY
+
+----
+======
+=====
++
+When a DNS lookup is performed, the domain name returns an A record for every server in the cluster, where each A record contains the IP address of the server.
 The configured server uses all the IP addresses from the A records to join or form a cluster.
 +
 [NOTE]
 ====
 The discovery port must be the same on all servers when using this configuration.
-If this is not possible, consider using the discovery type `SRV` instead.
+If this is not possible, consider using the discovery type `SRV`.
 ====
 
 `dbms.cluster.discovery.resolver_type=SRV`::
-With this configuration, the initial discovery members are resolved from _DNS SRV_ records to find the IP addresses/hostnames and discovery service ports to contact.
-The value of `dbms.cluster.discovery.endpoints` should be set to a single domain name and the port set to `0`.
-For example: `dbms.cluster.discovery.endpoints=cluster01.example.com:0`.
-The domain name should return a single SRV record when a DNS lookup is performed.
-The SRV record returned by DNS should contain the IP address or hostname, and the discovery port for the servers to be discovered.
-The configured server uses all the addresses from the SRV record to join or form a cluster.
+With this configuration, the initial discovery members are resolved from _DNS SRV_ records to find the IP addresses/hostnames and discovery service ports (v1) *or* cluster advertised ports (v2) to contact.
++
+The value of `dbms.cluster.discovery.endpoints` must be set to a single domain name and the port set to `0`.
+The domain name returns a single SRV record when a DNS lookup is performed.
+For example:
++
+[.tabbed-example]
+=====
+[role=include-with-discovery-service-v1 label--deprecated-5.23]
+======
+[source, parameters]
+----
+dbms.cluster.discovery.resolver_type=SRV
 
+server.discovery.advertised_address=server01.example.com:5000
+dbms.cluster.discovery.endpoints=cluster01.example.com:0
 
-[[clustering-discovery-k8s]]
-== Discovery in Kubernetes
+dbms.cluster.discovery.version=V1_ONLY
+----
+======
+[role=include-with-discovery-service-v2 label--new-5.23]
+======
+[source, parameters]
+----
+dbms.cluster.discovery.resolver_type=SRV
+
+server.cluster.advertised_address=server01.example.com:6000
+dbms.cluster.discovery.v2.endpoints=cluster01.example.com:0
+dbms.cluster.discovery.version=V2_ONLY
+----
+======
+=====
++
+The SRV record returned by DNS contains the IP address or hostname, and the cluster port for the servers to be discovered.
```

Comment:
So the original of this line is :
`
The SRV record returned by DNS should contain the IP address or hostname, and the discovery port for the servers to be discovered.
`

for v2 it'll be 
`
The SRV record returned by DNS should contain the IP address or hostname, and the **cluster** port for the servers to be discovered. 
`
So it might need to go into the tab or bold maybe? to make it clear of the difference